conda_smithy/templates/README.md.tmpl

```diff
@@ -2,31 +2,44 @@
 {#-
 # -*- mode: jinja -*-
 -#}
-{%- set license_url = package_about.license_url -%}
 
-About {{ package_name }}
-======{{ '=' * package_name|length }}
+About {{ package_name }}-feedstock
+======{{ '=' * package_name|length }}==========
 
-Home: {{ package_about.home }}
+Feedstock license: [BSD-3-Clause](https://github.com/{{ github.user_or_org }}/{{ github.repo_name }}/blob/{{ github.branch_name }}/LICENSE.txt)
 
-Package license: {% if license_url %}[{% endif %}{{ package_about.license }}{% if license_url %}]({{ license_url }}){% endif %}
+{% for output_name, about in package_about %}
+{%- set license_url = about.license_url -%}
 
-Feedstock license: [BSD-3-Clause](https://github.com/{{ github.user_or_org }}/{{ github.repo_name }}/blob/{{ github.branch_name }}/LICENSE.txt)
+About {{ output_name }}
+------{{ '-' * output_name|length }}
```

Comment:
Can you omit these two lines if `len(package_about)` is one?

Reply:
Done in commit d5cad79.